docs/DdnRelocatableShaderElf.md

```diff
@@ -137,9 +137,9 @@ top are the three "top-level" nodes. They are considered the top because the can
 be accessed directly. On the other hand, to access the descriptor for say
 descriptor set 0 and binding 2, the DescriptorTableVaPtr must be loaded first.
 
-Buffer descriptors are placed in the top-level are called "dynamic
-descriptors", and they are used to implement the feature where the application
-can modify the start offset of buffer without rebuilding the pipeline state.
+Buffer descriptors are placed in the top-level are called "dynamic descriptors",
```

Comment:
*that* are placed?

This change seems unrelated to what the commit message talks about.

Reply:
I ran a markdown formatter on the document, and it changed some of the other code.  I can undo that.